docs/guide/technology-choices/compute-comparison.md

```diff
@@ -13,12 +13,12 @@ The term *compute* refers to the hosting model for the computing resources that
 
 | Criteria | Virtual Machines | App Service | Service Fabric | Azure Functions | Azure Container Service | Cloud Services | Azure Batch |
 |----------|-----------------|-------------|----------------|-----------------|-------------------------|----------------|-------------|
-| Application composition | Agnostic | Applications | Services, guest executables | Functions | Containers | Roles | Scheduled jobs  |
+| Application composition | Agnostic | Applications, Linux Containers | Services, guest executables, containers | Functions | Linux Containers, Windows Containers (preview) | Roles | Scheduled jobs  |
 | Density | Agnostic | Multiple apps per instance via app plans | Multiple services per VM | No dedicated instances <a href="#note1"><sup>1</sup></a> | Multiple containers per VM | One role instance per VM | Multiple apps per VM |
-| Minimum number of nodes | 1 <a href="#note2"><sup>2</sup></a>  | 1 | 5 <a href="#note3"><sup>3</sup></a> | No dedicated nodes <a href="#note1"><sup>1</sup></a> | 3 | 2 | 1 <a href="#note4"><sup>4</sup></a> |
+| Minimum number of nodes | 1 <a href="#note2"><sup>2</sup></a>  | 1 | 5 <a href="#note3"><sup>3</sup></a>. Azure VM Scale Set supported | No dedicated nodes <a href="#note1"><sup>1</sup></a> | 3. Azure VM Scale Set not supported | 2 | 1 <a href="#note4"><sup>4</sup></a> |
 | State management | Stateless or Stateful | Stateless | Stateless or stateful | Stateless | Stateless or Stateful | Stateless | Stateless |
-| Web hosting | Agnostic | Built in | Self-host, IIS in containers | Not applicable | Agnostic | Built-in (IIS) | No |
-| OS | Windows, Linux | Windows, Linux (preview)  | Windows, Linux  (preview) | Not applicable | Windows,  Linux | Windows | Windows, Linux |
+| Web hosting | Agnostic | Built in | Self-host, IIS | Not applicable | Self-host, IIS | Built-in (IIS) | No |
```

Comment:
For ACS, I think we should leave this as "agnostic" - you can run anything (HAProxy, Nginx, Tomcat)

Reply:
Done.